### PR TITLE
Fix #1271 Add slack_file object to image block / block element classes

### LINKED ISSUE #1271: Add slack_file object to image block / block element classes
References:
* https://api.slack.com/reference/block-kit/blocks#image_fields
* https://api.slack.com/reference/block-kit/block-elements#image
* https://api.slack.com/reference/block-kit/composition-objects#slack_file

### Category (place an `x` in each of the `[ ]`)

* [ ] **bolt** (Bolt for Java)
* [ ] **bolt-{sub modules}** (Bolt for Java - optional modules)
* [ ] **slack-api-client** (Slack API Clients)
* [x] **slack-api-model** (Slack API Data Models)
* [ ] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [ ] **slack-app-backend** (The primitive layer of Bolt for Java)

### Requirements

Please make sure if this topic is specific to this SDK. For general questions/issues about Slack API platform or its server-side, could you submit questions at https://my.slack.com/help/requests/new instead. :bow:

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you agree to those rules.

### PR DESCRIPTION
**Pull request by seratch**
This pull request resolves #1271 

### Category (place an `x` in each of the `[ ]`)

* [ ] **bolt** (Bolt for Java)
* [ ] **bolt-{sub modules}** (Bolt for Java - optional modules)
* [ ] **slack-api-client** (Slack API Clients)
* [x] **slack-api-model** (Slack API Data Models)
* [x] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [ ] **slack-app-backend** (The primitive layer of Bolt for Java)

## Requirements

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you agree to those rules.

### CODE REVIEW

File: slack-api-client/src/test/java/test_with_remote_apis/methods/BlockKit_Test.java [hunk 1]
```diff
@@ -365,4 +370,46 @@ public void richTextMessage() throws Exception {
...
+            Thread.sleep(300L);
+            shares = client.filesInfo(r -> r.file(fileId)).getFile().getShares();
+        }
+        // Workaround to delete the previous message to share the file
+        String ts = shares.getPublicChannels().get(randomChannelId).get(0).getTs();
+        ChatDeleteResponse deletion = client.chatDelete(r -> r.channel(randomChannelId).ts(ts));
+        assertThat(deletion.getError(), is(nullValue()));
+        Thread.sleep(500L);
+
+        ChatPostMessageResponse response = client.chatPostMessage(r -> r
```
Comment (filmaj) [inline slack-api-client/src/test/java/test_with_remote_apis/methods/BlockKit_Test.java:401]: I suppose there is no way to validate this message correctly displays the image?

**Review by filmaj — APPROVED** [review]
Thanks for tackling this! The tests look extensive.

File: slack-api-client/src/test/java/test_with_remote_apis/methods/BlockKit_Test.java (hunk 1 above)
Comment (seratch) [reply slack-api-client/src/test/java/test_with_remote_apis/methods/BlockKit_Test.java:401]: The only approach would be to check if there are the blocks in the message property in the response, but it sounds repetitive to me. Validating whether there is no error code such as invalid_blocks in the response (like the current test code does) should be good enough this time.

**Comment by seratch** [comment]
Thanks for the quick review!

**Pull request merged**